playbooks/common/openshift-master/config.yml

```diff
@@ -166,16 +164,11 @@
       port: "{{ openshift.master.dns_port }}/tcp"
     - service: skydns udp
       port: "{{ openshift.master.dns_port }}/udp"
-    - service: Fluentd td-agent tcp
-      port: 24224/tcp
-    - service: Fluentd td-agent udp
-      port: 24224/udp
-    - service: pcsd
-      port: 2224/tcp
-    - service: Corosync UDP
-      port: 5404/udp
-    - service: Corosync UDP
-      port: 5405/udp
+  - role: os_firewall
+    os_firewall_allow:
+    - service: etcd embedded
+      port: 4001/tcp
+    when: embedded_etcd | bool
```

Comment:
We could check this as `groups.oo_etcd_to_config | default([]) | length == 0`. There's also an `openshift.master.embedded_etcd` fact which is equivalent and arrived at similarly.